sdk/src/azure/core/az_context.c

```diff
@@ -33,6 +35,8 @@ AZ_NODISCARD int64_t az_context_get_expiration(az_context const* context)
 AZ_NODISCARD az_result
 az_context_get_value(az_context const* context, void const* key, void const** out_value)
 {
+  _az_PRECONDITION_NOT_NULL(context);
```

Comment:
Also, out_value pointer can't be null since we dereference it to return the found value to the caller. Please add a test.
```C
az_PRECONDITION_NOT_NULL(*out_value);
```

Reply:
Agreed.  There is a much more extensive C++ suite